testgen/src/main/groovy/GenerateApiDocsTest.groovy

```diff
@@ -0,0 +1,54 @@
+package com.vivy.springdoc
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.TaskAction
+
+class GenerateApiDocsTest extends DefaultTask {
+
+    @Input
+    String openApiDocsTestOut;
+    @Input
+    String testPackage;
```

Comment:
i would fix it to some value - like com.vivy.openapi

Reply:
Ok, will do optional with predefined value.